services/director-v2/tests/unit/test_modules_dynamic_sidecar_volumes_resolver.py

```diff
@@ -52,21 +52,27 @@ def test_expected_paths(
     state_paths: List[Path],
     expect: Callable[[str, str], Dict[str, Any]],
 ) -> None:
-    path = Path("/inputs")
+    inputs_path = Path("/tmp/some/inputs")
```

Comment:
please use the tmp_folder fixture for this

Reply:
no need for the folder here, only the name

Reply:
yes but you are still mounting things from a folder that might not always exist in every distro right? tmp_folder fixture would prevent such problems

Reply:
You are correct. The director-v2 will never mount to that path. It is just used to send the information downstream to the dynamic-sidecar when it is started.
Also in the scope of this unit test nothing will get mounted as well.

Reply:
then a tip: you should use the faker if the path is totally irrelevant. it would make the intent clearer